main/getting-started/start-a-project.md

```diff
@@ -35,70 +32,36 @@ This presentation includes starting a project, but note an outdated detail:
 Use the [Agoric CLI](/guides/agoric-cli/commands.md) to fetch from a Dapp template
 and put it in a `demo` directory _not located in your `agoric-sdk` clone_:
 
-
 ```sh
 # Terminal 1
-# Don't use your agoric-sdk clone as the parent of the demo directory.
-# It doesn't have to be your home directory; any other directory will do.
+# Don't use your agoric-sdk as the parent of the demo directory.
 cd $HOME
-agoric init demo
+agoric init demo # use `agoric init $DIRNAME` with any name you like
+cd demo
+agoric install
+agoric start --verbose # `agoric start --reset` to start over
 ```
 
-The name `demo` is an arbitrary suggestion; in general,
-use `agoric init $DIRNAME` with any name you like.
+The name `demo` is an arbitrary suggestion. The default template is the [Fungible Faucet Dapp](https://github.com/Agoric/dapp-fungible-faucet).
 
-The default template is the [Fungible Faucet Dapp](https://github.com/Agoric/dapp-fungible-faucet).
 Learn more about the [available dapp templates](/dapps/dapp-templates.md).
 
-### Install the Agoric SDK in the Dapp
```

Comment:
This heading seems important to help the reader build a mental model of the parts of the system; in particular, that a dapp depends on the SDK but is a separate thing. Do you expect to explain that somewhere else? Or do you think we don't need it?

Reply:
i would argue that we do not need it as it is covered under the consolidated "Initialized `demo` from Dapp Template` section.

Reply:
The only mention I see in the revised version of that section is "Don't use your agoric-sdk as the parent of the demo directory." I don't see how that establishes in the readers mind that the SDK and the dapp are separate things, where one gets installed as a dependency of the other.

Reply:
i've amended the introduction into three sentences at the top of "start a project" to clarify the distinction... maybe that helps?